### PR TITLE
Enable Java unit test coverage report

### PR DESCRIPTION
**Pull request by zhongqiliang**
b/285169626

### CODE REVIEW

File: starboard/android/apk/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
```
Comment (kaidokert) [inline starboard/android/apk/gradle/wrapper/gradle-wrapper.properties:3]: This is pointing at `7.3.3` while above is at `7.2.2` - which one is correct and do they need to be aligned ?

Comment (zhongqiliang) [reply starboard/android/apk/gradle/wrapper/gradle-wrapper.properties:3]: Check the versions here https://developer.android.com/build/releases/past-releases/agp-7-2-0-release-notes#compatibility-7-2-0.

**Comment by oxve** [comment]
Where does the coverage data end up? I can help you add a build pipeline step that archives the coverage report and uploads the data to codecov.

File: starboard/android/apk/app/build.gradle [hunk 2]
```diff
@@ -149,16 +149,16 @@ android {
...
-            assets.srcDir "${buildDir}/intermediates/cxx/devel/cobalt_content"
+            assets.srcDir "${cobaltContentDir}"
         }
         qa {
-            assets.srcDir "${buildDir}/intermediates/cxx/qa/cobalt_content"
+            assets.srcDir "${cobaltContentDir}"
         }
         release {
-            assets.srcDir "${buildDir}/intermediates/cxx/gold/cobalt_content"
+            assets.srcDir "${cobaltContentDir}"
```
Comment (oxve) [inline starboard/android/apk/app/build.gradle:161]: What are these changes for?

Comment (zhongqiliang) [reply starboard/android/apk/app/build.gradle:161]: To work around with the problem caused by gradle upgrade.

**Comment by zhongqiliang** [comment]
Commands to generate the coverage report.

cd ./starboard/android/apk
./gradlew createDebugUnitTestCoverageReport
The reports are under

out/android_studio/gradle/build/app/reports/coverage/test/debug/index.html
out/android_studio/gradle/build/app/reports/coverage/test/debug/report.xml

**Review by oxve — APPROVED** [review]

**Pull request merged**